photon-api/src/main/scala/com/linkedin/photon/ml/data/RandomEffectDataset.scala

```diff
@@ -287,27 +286,24 @@ object RandomEffectDataset {
       randomEffectDataConfiguration,
       randomEffectPartitioner)
     val projectedGroupedActiveData = generateProjectedActiveData(unfilteredActiveData, unfilteredProjectors)
-    val projectedUnfilteredActiveData = featureSelectionOnActiveData(
-      projectedGroupedActiveData,
-      randomEffectDataConfiguration.numFeaturesToSamplesRatioUpperBound)
```

Comment:
Forgot to remove the `numFeaturesToSamplesRatioUpperBound` from the data configuration